configure.ac

```diff
@@ -3477,6 +3477,19 @@ CheckVirtualJoystick()
     fi
 }
 
+dnl Set up the Translator joystick driver.
+CheckTranslatorJoystick()
+{
+    AC_ARG_ENABLE(joystick-translator,
+[AS_HELP_STRING([--enable-joystick-translator], [enable translator joystick [default=yes]])],
+                  , enable_joystick_translator=yes)
+    if test x$enable_joystick = xyes -a x$enable_joystick_translator = xyes; then
+        AC_DEFINE(SDL_JOYSTICK_TRANSLATOR, 1, [ ])
+        SOURCES="$SOURCES $srcdir/src/joystick/translator/*.c"
+        have_joystick_translator=yes
```

Comment:
This `have_joystick_translator=yes` line isn't used, can be removed.
OK with that change, from my part. @slouken should also review.